infra/base-images/base-runner/bad_build_check

```diff
@@ -62,78 +62,69 @@ function check_startup_crash {
     CHECK_PASSED=1
   fi
 
-  if (( $CHECK_PASSED == 0 )); then
-    echo "BAD BUILD: the fuzzer seems to have either startup crash or exit."
-    exit 1
+  if [ "$CHECK_PASSED" -eq "0" ]; then
+    echo "BAD BUILD: the fuzzer seems to have either startup crash or exit." >> $LOG_PATH_FOR_BROKEN_TARGET
+    BAD_BUILD_CHECK_FAILED=1
   fi
 }
 
 # Mixed sanitizers check for ASan build.
 function check_asan_build {
   local FUZZER=$1
-  local ASAN_CALLS=$2
-  local MSAN_CALLS=$3
-  local UBSAN_CALLS=$4
-
-  # Perform all the checks for more verbose error message.
-  local CHECK_FAILED=0
+  local LOG_PATH_FOR_BROKEN_TARGET=$2
+  local ASAN_CALLS=$3
+  local MSAN_CALLS=$4
+  local UBSAN_CALLS=$5
 
+  # Perform all the checks for more detailed error message.
   if (( $ASAN_CALLS < 1000 )); then
```

Comment:
Move these threshold 1000, 1000, 250 to start of file and easily configurable.

Reply:
Good point, done!